### ISSUE TITLE
'i' button present with the link edit field is not accessible using the keyboard as keyboard focus is not going on it.

### ISSUE THREAD

**Issue by ElinorW** [microsoftgraph/microsoft-graph-explorer-v4#1372]
**Test Environment:**
OS Version: Windows 11 Build 22504.1010
Browser: Edge Dev Version 98.0.1093.6 (Official build) dev (64-bit)
URL:Graph Explorer - Microsoft Graph
Screen Reader: Narrator
Username: hcltester200@outlook.com/hcltester800@outlook.com

**Repro Steps:**

1. Open above URL in Edge browser.
2. Login with valid credential.
3. Navigate to the 'i' button present beside link edit field using tab key.
4. Observe if keyboard focus is going on it.

**Actual Result:**
'i' button present with the link edit field is not accessible using the keyboard as keyboard focus is not going on it.

**Expected Result:**
'i' button present with the link edit field should be accessible with the keyboard as keyboard focus should go on it using the tab key.

**User Impact:** 
Keyboard users will be impacted as they will not be able to access the specified control using the keyboard.

**MAS Reference:**
[Keyboard - Liquid (microsoft.com)](https://liquid.microsoft.com/Web/Object/Read/MS.Accessibility/Requirements/02.01.01)
